autograder/tests/test_submitty_autograding_shipper.py

```diff
@@ -0,0 +1,152 @@
+import os
+import json
+import shutil
+import sys
+import unittest
+import contextlib
+import copy
+
+# Done to avoid making the base autograder directory a module.
+# This import must be done inside of the class for the mock to take affect
+sys.path.append("..")
```

Comment:
```suggestion
```

This isn't necessary as long as we run the tests from within the `autograding/` directory which, judging from [.travis.yml](https://github.com/Submitty/Submitty/blob/master/.travis.yml#L155-L157), we are doing. Running `coverage -m unittest discover` from `autograding/` runs fine on my VM without this line.

Reply:
Removed